database/plugin/blob/aws/plugin.go

```diff
@@ -21,12 +21,12 @@ import (
 )
 
 var (
-	cmdlineOptions struct {
-		bucket string
-		region string
-		prefix string
+	CmdlineOptions struct {
+		Bucket string
+		Region string
+		Prefix string
 	}
-	cmdlineOptionsMutex sync.RWMutex
+	CmdlineOptionsMutex sync.RWMutex
```

Comment:
This mutex is only referenced inside this package and should not be exported.

Separately, is this mutex even useful? There doesn't seem much change of race conditions unless we plan to initialize multiple copies of this plugin concurrently.